### PR TITLE
[SYCLomatic-test] Add Help Function for Sort Headers

### PR DESCRIPTION
**Pull request by abhilash1910**
Add block sort test Following PR #1562 https://github.com/oneapi-src/SYCLomatic/pull/1562
cc @yihanwg

### CODE REVIEW

File: help_function/src/onedpl_test_group_sort.cpp [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+      return false;
+    }
+  }
+  std::cout << "test_sort_blocked_to_striped pass\n";
+  return true;
+}
+
+int main() {
+  return !(test_sort_blocked() && test_sort_blocked_to_striped());
+}
```
Comment (yihwang-nv) [inline help_function/src/onedpl_test_group_sort.cpp]: Please add new line at the end of this file.

File: help_function/src/onedpl_test_group_sort.cpp [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+bool test_sort_blocked_to_striped() {
+  sycl::queue q;
+  int data[512];
+  for (int i = 0; i < 512; i++) data[i] = i;
+
+  sycl::buffer<int, 1> buffer(data, 512);
+
+  q.submit([&](sycl::handler &h) {
+    using group_sort = dpct::group::sort<int, 4>;
```
Comment (yihwang-nv) [inline help_function/src/onedpl_test_group_sort.cpp]:
```suggestion
    using group_sort = dpct::group::radix_sort<int, 4>;

```

File: help_function/src/onedpl_test_group_sort.cpp [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+  for (int i = 0; i < 128; i++) {
+    data[4 * i + 0] = i;
+    data[4 * i + 1] = i + 1 * 128;
+    data[4 * i + 2] = i + 2 * 128;
+    data[4 * i + 3] = i + 3 * 128;
+  }
+
+  sycl::buffer<int, 1> buffer(data, 512);
+  q.submit([&](sycl::handler &h) {
+    using group_sort = dpct::group::sort<int, 4>;
```
Comment (yihwang-nv) [inline help_function/src/onedpl_test_group_sort.cpp]:
```suggestion
    using group_sort = dpct::group::radix_sort<int, 4>;

```

File: help_function/src/onedpl_test_group_sort.cpp (hunk 2 above)
Comment (abhilash1910) [reply help_function/src/onedpl_test_group_sort.cpp]: Thanks fixed.

**Comment by yihwang-nv** [comment]
Thank you work on this! Could you please check the test inputs/outputs, I tried to run this patch locally but run failed.

**Comment by abhilash1910** [comment]
@yihanwg, @zhimingwang36   test seems fixed now, could you please review ? Thanks

**Review by yihwang-nv — APPROVED** [review]
LGTM

**Review by ShengchenJ — APPROVED** [review]

**Pull request merged**